ifrs17-template/Database/MigrationAndScaffolding/ReadMe.ipynb

```diff
@@ -0,0 +1,117 @@
+{
+    "metadata": {
+        "authors": [],
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Brief Explanation of Migration"
```

Comment:
Start the NB with a big Title. You can copy the code for the markdown from other NBs